jobboard/tests/model_function_tests.py

```diff
@@ -3,6 +3,7 @@
 from student.models import Major
 from datetime import date
 
+
```

Comment:
Can you remove this? No need to change this file if you don't add anything to it, it might cause unnecessary conflicts with different PRs